src/player_data.h

```diff
@@ -143,11 +150,11 @@ struct PlayerInfo {
     unsigned char allocflags;
     unsigned char field_1;
     unsigned char boxsize; //field_2 seems to be used in DK, so now renamed and used in KeeperFX
-    unsigned char field_3;
+    unsigned char field_3; // Additional Cursor/Current Subtile Information?
     unsigned char input_crtr_control;
     unsigned char input_crtr_query;
     unsigned char flgfield_6;
-    unsigned char *field_7;
+    unsigned char *palette_B; // ?
```

Comment:
remove //?

Reply:
I'm aiming to do better than palette_A and palette_B as field names - but either way I'll tidy this up before finishing the PR

Reply:
resolved in 50548bb